src/CONST.js

```diff
@@ -332,6 +321,19 @@ const CONST = {
\ud83d\udebd|\ud83e\udea0|\ud83d\udebf|\ud83d\udec1|\ud83e\udea4|\ud83e\ude92|\ud83e\uddf4|\ud83e\uddf7|\ud83e\uddf9|\ud83e\uddfa|\ud83e\uddfb|\ud83e\udea3|\ud83e\uddfc|\ud83e\udea5|\ud83e\uddfd|\ud83e\uddef|\ud83d\uded2|\ud83d\udeac|\ud83e\udea6|\ud83d\uddff|\ud83e\udea7|\ud83c\udfe7|\ud83d\udeae|\ud83d\udeb0|\ud83d\udeb9|\ud83d\udeba|\ud83d\udebb|\ud83d\udebc|\ud83d\udebe|\ud83d\udec2|\ud83d\udec3|\ud83d\udec4|\ud83d\udec5|\ud83d\udeb8|\ud83d\udeab|\ud83d\udeb3|\ud83d\udead|\ud83d\udeaf|\ud83d\udeb1|\ud83d\udeb7|\ud83d\udcf5|\ud83d\udd1e|\ud83d\udd03|\ud83d\udd04|\ud83d\udd19|\ud83d\udd1a|\ud83d\udd1b|\ud83d\udd1c|\ud83d\udd1d|\ud83d\uded0|\ud83d\udd49|\ud83d\udd4e|\ud83d\udd2f|\ud83d\udd00|\ud83d\udd01|\ud83d\udd02|\ud83d\udd3c|\ud83d\udd3d|\ud83c\udfa6|\ud83d\udd05|\ud83d\udd06|\ud83d\udcf6|\ud83d\udcf3|\ud83d\udcf4|\ud83d\udcb1|\ud83d\udcb2|\ud83d\udd31|\ud83d\udcdb|\ud83d\udd30|\ud83d\udd1f|\ud83d\udd20|\ud83d\udd21|\ud83d\udd22|\ud83d\udd23|\ud83d\udd24|\ud83c\udd70|\ud83c\udd8e|\ud83c\udd71|\ud83c\udd91|\ud83c\udd92|\ud83c\udd93|\ud83c\udd94|\ud83c\udd95|\ud83c\udd96|\ud83c\udd7e|\ud83c\udd97|\ud83c\udd7f|\ud83c\udd98|\ud83c\udd99|\ud83c\udd9a|\ud83c\ude01|\ud83c\ude02|\ud83c\ude37|\ud83c\ude36|\ud83c\ude2f|\ud83c\ude50|\ud83c\ude39|\ud83c\ude1a|\ud83c\ude32|\ud83c\ude51|\ud83c\ude38|\ud83c\ude34|\ud83c\ude33|\ud83c\ude3a|\ud83c\ude35|\ud83d\udd34|\ud83d\udfe0|\ud83d\udfe1|\ud83d\udfe2|\ud83d\udd35|\ud83d\udfe3|\ud83d\udfe4|\ud83d\udfe5|\ud83d\udfe7|\ud83d\udfe8|\ud83d\udfe9|\ud83d\udfe6|\ud83d\udfea|\ud83d\udfeb|\ud83d\udd36|\ud83d\udd37|\ud83d\udd38|\ud83d\udd39|\ud83d\udd3a|\ud83d\udd3b|\ud83d\udca0|\ud83d\udd18|\ud83d\udd33|\ud83d\udd32|\ud83c\udfc1|\ud83d\udea9|\ud83c\udf8c|\ud83c\udff4|\ud83c\udff3|\ud83c\udffb|\ud83c\udffc|\ud83c\udffd|\ud83c\udffe|\ud83c\udfff|\u263a|\u2639|\u2620|\u2763|\u2764|\u270b|\u270c|\u261d|\u270a|\u270d|\u26f7|\u26f9|\u2618|\u2615|\u26f0|\u26ea|\u26e9|\u26f2|\u26fa|\u2668|\u26fd|\u2693|\u26f5|\u26f4|\u2708|\u231b|\u23f3|\u231a|\u23f0|\u23f1|\u23f2|\u2600|\u2b50|\u2601|\u26c5|\u26c8|\u2602|\u2614|\u26f1|\u26a1|\u2744|\u2603|\u26c4|\u2604|\u2728|\u26bd|\u26be|\u26f3|\u26f8|\u2660|\u2665|\u2666|\u2663|\u265f|\u26d1|\u260e|\u2328|\u2709|\u270f|\u2712|\u2702|\u26cf|\u2692|\u2694|\u2699|\u2696|\u26d3|\u2697|\u26b0|\u26b1|\u267f|\u26a0|\u26d4|\u2622|\u2623|\u2b06|\u2197|\u27a1|\u2198|\u2b07|\u2199|\u2b05|\u2196|\u2195|\u2194|\u21a9|\u21aa|\u2934|\u2935|\u269b|\u2721|\u2638|\u262f|\u271d|\u2626|\u262a|\u262e|\u2648|\u2649|\u264a|\u264b|\u264c|\u264d|\u264e|\u264f|\u2650|\u2651|\u2652|\u2653|\u26ce|\u25b6|\u23e9|\u23ed|\u23ef|\u25c0|\u23ea|\u23ee|\u23eb|\u23ec|\u23f8|\u23f9|\u23fa|\u23cf|\u2640|\u2642|\u26a7|\u2716|\u2795|\u2796|\u2797|\u267e|\u203c|\u2049|\u2753|\u2754|\u2755|\u2757|\u3030|\u2695|\u267b|\u269c|\u2b55|\u2705|\u2611|\u2714|\u274c|\u274e|\u27b0|\u27bf|\u303d|\u2733|\u2734|\u2747|\u00a9|\u00ae|\u2122|\u2139|\u24c2|\u3297|\u3299|\u26ab|\u26aa|\u2b1b|\u2b1c|\u25fc|\u25fb|\u25fe|\u25fd|\u25aa|\u25ab)/g,
```

Comment:
I thought this is going to be a short one. :laughing:

Reply:
😄 There're more short versions, but they don't really work for our use case. 

The reason they're shorter is that they work with unicode ranges, but by doing this a composed emoji (a emoji that is made from a sequence of other emojis, like 👩‍👩‍👧‍👦 =  👨👩👧👦) will match with all it's parts. So

'👨👩👧👦'.match(regex) will give the same result as '👩‍👩‍👧‍👦'.match(regex) and we wouldn't be able to differentiate between a single (but composed emoji) and a emoji sequence.

Reply:
Check for length. nad No of surrogate pairs.